PWGHF/TableProducer/candidateCreatorToXiPi.cxx

```diff
@@ -93,6 +94,13 @@ struct HfCandidateCreatorToXiPi {
 
   void init(InitContext const&)
   {
+    if (doprocessWithCascades == true && doprocessWithTrackedCascades == true) {
+      LOGF(fatal, "Cannot enable processWithCascades and processWithTrackedCascades at the same time. Please choose one.");
+    }
+    if (doprocessWithCascades == false && doprocessWithTrackedCascades == false) {
+      LOGF(fatal, "Please enable at least one process function, either processWithCascades or processWithTrackedCascades.");
+    }
```

Comment:
```suggestion
    if (doprocessWithCascades && doprocessWithTrackedCascades) {
      LOGF(fatal, "Cannot enable processWithCascades and processWithTrackedCascades at the same time. Please choose one.");
    }
    if (!doprocessWithCascades && !doprocessWithTrackedCascades) {
      LOGF(fatal, "Please enable at least one process function, either processWithCascades or processWithTrackedCascades.");
    }
```
- Do not use ` == true/false`. It has no effect and just makes code longer.
- Instead of two checks, you can just have one check with `std::accumulate`.